### DISCUSSION TITLE
Can we use this also for IOS and Android Emulators

### DISCUSSION THREAD

**Discussion by inluxc** [NetrisTV/ws-scrcpy#118] [Q&A] [answered]
Is there a way to also load IOS and Android Emulators?

**Comment by drauggres** [comment, marked as answer]
> IOS

No, physical device only.

> Android Emulators

Works fine with `proxy over adb`: https://github.com/NetrisTV/ws-scrcpy#known-issues

**Reply by Harishvaluepitch** [reply]
how to configue with ios on windows?

**Reply by drauggres** [reply]
See https://github.com/NetrisTV/ws-scrcpy/discussions/200

**Comment by Harishvaluepitch** [comment]
Thank you so much man, i appreciate that.

Thanks,
Harish.
